app/views/insured/product_shoppings/_plan_details.html.slim

```diff
@@ -14,7 +14,7 @@
               = l10n("premium_value_plan").to_s.upcase
           - if product.try(:is_standard_plan)
             span.standard-plan-indicator
-              i.fas.fa-lg.fa-square.mr-1 data-toggle="tooltip" data-placement="top" data-container="body" title="#{l10n('insured.plan_shoppings.standard_plan_title_info')}"
+              i.fas.fa-lg.fa-bookmark.mr-1 data-toggle="tooltip" data-placement="top" data-container="body" title="#{l10n('insured.plan_shoppings.standard_plan_title_info')}"
```

Comment:
This seemed ok as it was, to me. [Requirements doc](https://docs.google.com/document/d/1c0k72bo6NqR8E2zqMmkl9DaO9URuZpn-Qf3EOhu8LuI/edit#heading=h.b0v42fdum4kl) shows that when the feature is enabled, std and pvp should both use a square. The toggled-off, bookmark-symbol case seems to be covered by https://github.com/health-connector/enroll/pull/2757/files#diff-188efa87547f1438241c625f9a6e96fd576ba5de9b87b7c46503272394cf24d2R22 below. But I might be referencing old info?